package.json

```diff
@@ -231,6 +231,24 @@
           "default": false,
           "description": "Enable/disable XML CodeLens."
         },
+        "xml.preferences.showSchemaDocumentationType": {
+          "type": "string",
+          "default": "all",
+          "enum": [
+            "documentation",
+            "appinfo",
+            "all",
+            "none"
+          ],
+          "markdownEnumDescriptions": [
+            "Documentation comes from the `documentation` element.",
```

Comment:
I will add `xs:` for each element like` xs:documentation`

Reply:
Thanks, fixed